compiler-rt/lib/sanitizer_common/sanitizer_stacktrace_printer.cpp

```diff
@@ -12,13 +12,28 @@
 
 #include "sanitizer_stacktrace_printer.h"
 
+#include "sanitizer_common.h"
 #include "sanitizer_file.h"
 #include "sanitizer_flags.h"
 #include "sanitizer_fuchsia.h"
 
 namespace __sanitizer {
 
-const char *StripFunctionName(const char *function) {
+StackTracePrinter *StackTracePrinter::stacktrace_printer_;
```

Comment:
Please remove these from header and just use

`static StackTracePrinter *stacktrace_printer;`

Even better GetOrInit  function local one

Reply:
made both static local to the function